### PR TITLE
Style updates

### LINKED ISSUE #310: Add `void` theme
https://nbviewer.org/github/JetBrains/lets-plot-kotlin/blob/master/docs/examples/jupyter-notebooks/f-4.4.2/theme_void.ipynb

### LINKED ISSUE #323: Add `margin` text parameter
https://nbviewer.org/github/JetBrains/lets-plot-kotlin/blob/master/docs/examples/jupyter-notebooks/text_margins.ipynb

### PR DESCRIPTION
**Pull request by AndreiKingsley**
* close #310
* close #323
* move custom style creation to the `Style` companion

### CODE REVIEW

File: kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/feature/Layout.kt [hunk 1]
```diff
@@ -7,9 +7,9 @@ package org.jetbrains.kotlinx.kandy.letsplot.feature
 import org.jetbrains.kotlinx.kandy.dsl.internal.PlotContext
 import org.jetbrains.kotlinx.kandy.ir.feature.FeatureName
 import org.jetbrains.kotlinx.kandy.ir.feature.PlotFeature
-import org.jetbrains.kotlinx.kandy.letsplot.theme.CustomTheme
-import org.jetbrains.kotlinx.kandy.letsplot.theme.Flavor
-import org.jetbrains.kotlinx.kandy.letsplot.theme.Theme
+import org.jetbrains.kotlinx.kandy.letsplot.style.CustomStyle
+import org.jetbrains.kotlinx.kandy.letsplot.style.Theme
```
Comment (zaleslaw) [inline kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/feature/Layout.kt:11]: what is style.Theme?

File: kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/jupyter/JupyterConfig.kt [hunk 2]
```diff
@@ -7,6 +7,18 @@ package org.jetbrains.kotlinx.kandy.letsplot.jupyter
 import org.jetbrains.kotlinx.kandy.util.context.SelfInvocationContext
 
 public class JupyterConfig : SelfInvocationContext {
-    public var applyColorScheme: Boolean = true
+    /**
+     * Is IDEA theme applied to plots in the notebook outputs rendered with Swing.
+     *
+     * Enabled by default.
+     */
+    public var themeApplied: Boolean = true
```
Comment (zaleslaw) [inline kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/jupyter/JupyterConfig.kt:15]: isThemeApplied

File: kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/jupyter/JupyterConfig.kt [hunk 3]
```diff
@@ -7,6 +7,18 @@ package org.jetbrains.kotlinx.kandy.letsplot.jupyter
...
+
+    /**
+     * Is Java Swing applied to plots in the notebook outputs.
+     * Otherwise, web (HTML + JS) rendering is used.
+     *
+     * Enabled by default.
+     */
     public var swingEnabled: Boolean = true
```
Comment (zaleslaw) [inline kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/jupyter/JupyterConfig.kt:23]: isSwingEnabled

**Review by zaleslaw — APPROVED** [review]
Cool, now it's much more consistent.
@AndreiKingsley I've just suggested renaming some variables according to variable naming standards for booleans

File: kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/feature/Layout.kt (hunk 1 above)
Comment (AndreiKingsley) [reply kandy-lets-plot/src/main/kotlin/org/jetbrains/kotlinx/kandy/letsplot/feature/Layout.kt:11]: Because `Theme` is in the `style` package)

**Review by devcrocod — APPROVED** [review]

**Pull request merged**